PWG/EMCAL/config/EMCalSampleConfig.yaml

```diff
@@ -6,6 +6,7 @@ CellBadChannel:
     enabled: true
 CellTimeCalib:
     enabled: true
+    doMergedBCs: false # flag to use one histogram for the time calibration instead of four for all BCs
```

Comment:
This needs to go in the default YAML file, `AliEmcalCorrectionConfiguration.yaml`. Please also match the comment indentation in that file.

Have you tested the code since making these changes? Because as it stands, if settings are not in the mentioned file, the CorrFW should fail to configure

Reply:
Hi Raymond,

I did all the modifications. I didn't make any test after removing the old setters. But you are right, it has to be tested before merging.

Reply:
Okay, thanks. Everything now looks fine from my perspective, up to the need for testing